Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.yml

```diff
@@ -362,6 +383,93 @@ script:
     - contextPath: TenableIO.Scan.Status
       description: The status of the scan specified.
       type: string
+  - arguments:
+    - default: false
+      description: The ID of the scan. Comma separated values.
+      isArray: true
+      name: scanId
+      required: true
+      secret: false
+    deprecated: false
+    description: Pauses all scans imputted as an array
+    execution: false
+    name: tenable-io-pause-scan
+    outputs:
+    - contextPath: TenableIO.Scan.Id
+      description: The unique id of the scan.
+      type: Unknown
+    - contextPath: TenableIO.Scan.Status
+      description: The status of the scan (completed, aborted, imported, pending,
+        running, resuming, canceling, cancelled, pausing, paused, stopping, stopped).
+      type: Unknown
+  - arguments:
+    - default: false
+      description: Scan ID to be resumed
+      isArray: true
+      name: scanId
+      required: true
+      secret: false
+    deprecated: false
+    description: Resume all scans inputted as an array. Comma separated values.
+    execution: false
+    name: tenable-io-resume-scan
+    outputs:
+    - contextPath: TenableIO.Scan.Id
+      description: The unique id of the scan.
+      type: Unknown
+    - contextPath: TenableIO.Scan.Status
+      description: The status of the scan (completed, aborted, imported, pending,
+        running, resuming, canceling, cancelled, pausing, paused, stopping, stopped).
+      type: Unknown
+  - arguments:
+    - default: false
+      description: Payload for REST call
+      isArray: true
```

Comment:
Could you explain why this is an array?

Reply:
Hi @guykeller, I am looking at the code to determine why it is an array.

Reply:
Done
Update 02.23.2021 1034 HRS EST:
Removed lines from code